app/views/medical_records/_form.html.erb

```diff
@@ -1,19 +1,20 @@
+<div class="container">
 <%= form_with(model: [medical_record.patient, medical_record]) do |form| %>
   <% if medical_record.errors.any? %>
     <div style="color: red">
       <h2><%= pluralize(medical_record.errors.count, "error") %> prohibited this medical_record from being saved:</h2>
 
       <ul>
-        <% medical_record.errors.each do |error| %>
-          <li><%= error.full_message %></li>
+        <% medical_record.errors.full_messages.each do |error| %>
+          <li><%= error.html_safe %></li>
         <% end %>
       </ul>
     </div>
   <% end %>
-
   <div>
     <%= form.label :patient_id, style: "display: block" %>
-    <%= form.text_field :patient_id %>
+    <p><%= medical_record.patient_id %></p>
+    <%= form.hidden_field :patient_id %>
   </div>
```

Comment:
You cannot close a tag after its parent tag.

Bad:
```
<div>
  <form>
</div>
 </form>
```

Good
```
<div>
  <form>
  </form>
</div>
```

Reply:
entendido